dp-terraform/helm/rhacs-terraform/templates/emailsender.yaml

```
@@ -36,6 +36,17 @@ spec:
             - name: AUTH_CONFIG_FROM_KUBERNETES
               value: "true"
           {{- end }}
+            # Reusing fleetshardSync.aws.region here since the Values file defines multiple
+            # aws region for different components and the emailsender should always use the same as FS
+            - name: AWS_REGION
+              value: {{ .Values.fleetshardSync.aws.region }}
```

@kovayur @kurlov I thought it's ok to reuse here, if not I can make the addon change to have this as dedicated parameter for emailsender.
```suggestion
              value: {{ .Values.fleetshardSync.aws.region }}
```

---

I am fine with reusing. It is highly unlikely that regions are different. But I can add a new field to addon if necessary